gen-web/src/index.js

```diff
@@ -226,7 +256,7 @@
 
       setTimeout(() => {
         try {
-          filesaver.saveAs(configFileBlob, genConfigFileName(deviceNumber, deviceID))
+          filesaver.saveAs(configFileBlob, genConfigFileName(HOLO_PORT_STARTING_DEVICE_NUMBER, deviceID))
```

Comment:
This may need to be reverted. else it will always use the deviceNumber as 0

Reply:
Now genConfigFileName doesn't use the device number to generate the file name.